Gems/LyShine/Code/Include/LyShine/Draw2d.h

```diff
@@ -582,7 +582,7 @@ class Draw2dHelper
     //! Helper to get the default IDraw2d interface
     static CDraw2d* GetDefaultDraw2d()
     {
-        if (gEnv && gEnv->pLyShine) // LYSHINE_ATOM_TODO - remove pLyShine and use bus interface
+        if (gEnv && gEnv->pLyShine) // [LYSHINE_ATOM_TODO][LYN-8716] Remove LyShine global interface pointer from legacy global environment
```

Comment:
Does this one need a GHI number?

Reply:
Converted to GHI (except for one Atom issue that was already in code that I'll follow up on).